PowerPointLabs/PowerPointLabs/Utils/FileDir.cs

```diff
@@ -139,6 +139,16 @@ public static bool MoveFolder(string oldPath, string newPath)
             return CopyFolder(oldPath, newPath) && DeleteFolder(oldPath);
         }
 
+        public static string GetTemporaryPngFilePath()
+        {
+            string tempDirectory = Path.GetTempPath();
+            Directory.CreateDirectory(tempDirectory);
+
+            string randomFileName = "pptlabs_" + Path.GetRandomFileName();

```

Comment:
Well, it's not a necessity, but there are a bunch of other files in the `AppData/Local/Temp` folder that do not necessarily belong to PPTLabs, and it would be good to differentiate. In fact, it seems that PictureSlidesLab also saves some temp files here under the folder name `pptlabs_pictureSlidesLab`, so it's probably good to standardise.